recipes/jupyter_conda/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jupyter_conda" %}
+{% set version = "2.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fb429fb3328306d434debcba67c33183934c08c45aa238eaf0584ca868428109
+
+build:
+  skip: True  # [py2k]
```

Comment:
Would drop the `skip` and replace with `noarch: python`.

Reply:
I don't like `noarch` because byte code compilation is not optimized in conda. The installation of new environment can be dramatically slow down because of it. So I prefer to avoid using it.

Regarding `skip`, see reply to your first comment.

Reply:
Well, 4 python files with all in all 779 lines of code, not a big deal having those compiled on environment creation. Noarch is also 1 CI run vs 2x3 CI runs (Python 3.6/3.7, win64, osx64, linux64).